Doc/reference/lexical_analysis.rst

```diff
@@ -582,7 +582,7 @@ Standard C.  The recognized escape sequences are:
 +-------------------------+---------------------------------+-------+
 | ``\v``                  | ASCII Vertical Tab (VT)         |       |
 +-------------------------+---------------------------------+-------+
-| :samp:`\\{ooo}`         | Character with octal value      | (2,4) |
+| :samp:`\\\\{ooo}`       | Character with octal value      | (2,4) |
 |                         | *ooo*                           |       |
```

Comment:
Is this saying that all three `ooo` characters are the octal value? That `\777` is the octal value `777`?

Or does the first `o` indicate that it's octal (compare the `x` for hexadecimal below) and the last two are the value? That `\o77` is the octal value `77`?

So should it be something like this?

```suggestion
| :samp:`\\\\{ooo}`       | Character with octal value      | (2,4) |
|                         | *oo*                            |       |
```

Or with another character for the sample to make it clearer? For example:

```suggestion
| :samp:`\\o{xx}`         | Character with octal value *xx* | (2,4) |
```

Reply:
I think it is all three:

```python
assert chr(0o123) == "\123"
```